### PR TITLE
Use the latest SDC pattern library for Response Operations Social UI 

### PR DESCRIPTION
**Pull request by jrbarnes9**
# Motivation and Context

Update Response Operations Social UI service to use components and patterns from the [SDC pattern library.](https://sdc-global-design-patterns.netlify.com/index.html)

# What has changed
- CSS removed that is also present in the pattern library
- Minimised about of required local css
- Removed all unused JS
- Removed unused templates
- A number of templates refactored to use pattern library markup
- Improved code generation UI to display new code in panel at top of reloaded page, rather than out of view at the bottom
- Tidied up indentation and whitespace in templates
- Updated some page titles to be more task centric and follow a consistent pattern across the service.

# How to test?
- Ensure the response-ops-social repo will run
- Ensure the response-ops-social repo units tests pass
- Run through all SEL UI user journeys to make sure that no UI regressions have occurred, and no journeys now fail.

# Links
https://trello.com/c/yCnRVGM8/326-update-sel-ui-to-use-the-pattern-library-cdn

### CODE REVIEW

**Comment by jrbarnes9** [comment]
[Update SEL UI to use the pattern library CDN](https://trello.com/c/yCnRVGM8/326-update-sel-ui-to-use-the-pattern-library-cdn)

File: response_operations_social_ui/templates/social-view-case-details.html [hunk 1]
```diff
@@ -1,74 +1,76 @@
...
+    </div>
+    {% endif %}
+  </dd>
+  </dl>
+  <div class="case-details u-mt-m">
+    <section class="social-case-section u-mt-l">
+      <h2 class="neptune">Property address</h2>
+      <div class="address" id="case-attributes">
+        {% for name in displayed_attributes %}
+          <div class="case-address__{{ name|lower }}" id="case-{{ name|lower|replace('_', '-') }}">
```
Comment (AdamHawtin) [inline response_operations_social_ui/templates/social-view-case-details.html]: The changes here are making the address all caps. If this is intentional, the acceptance tests will need updating before this goes in.

File: response_operations_social_ui/templates/partials/footer.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<footer class="footer"></footer>
```
Comment (boxadesign) [inline response_operations_social_ui/templates/partials/footer.html]: Do we need this?

File: response_operations_social_ui/templates/social-view-case-details.html (hunk 1 above)
Comment (jrbarnes9) [reply response_operations_social_ui/templates/social-view-case-details.html]: There's no written guidance on using the address component, so I'll check with eQ.  Should the acceptance tests be case-sensitive?

File: response_operations_social_ui/templates/partials/footer.html (hunk 2 above)
Comment (jrbarnes9) [reply response_operations_social_ui/templates/partials/footer.html]: The intention is to add the internal service footer at some point.  I think even a blank footer is useful to reassure users they've hit the end of the page.

File: response_operations_social_ui/templates/social-view-case-details.html (hunk 1 above)
Comment (jrbarnes9) [reply response_operations_social_ui/templates/social-view-case-details.html]: @AdamHawtin  It turns out the pattern needs to be deprecated - so I'll amend this.

Comment (AdamHawtin) [reply response_operations_social_ui/templates/social-view-case-details.html]: The acceptance tests just string match on the data it was set up with. I believe in the past our philosophy was always display the address as it comes to us/is stored.

Comment (jrbarnes9) [reply response_operations_social_ui/templates/social-view-case-details.html]: @AdamHawtin Can you check the amendments?

**Pull request merged**